lib/fsm/utils.go

```diff
@@ -41,9 +41,54 @@ func CanRollback(plan *storage.OperationPlan, phaseID string) error {
 		return trace.BadParameter(
 			"phase %q has already been rolled back", phase.ID)
 	}
+
+	leafPhases := plan.GetLeafPhases()
+
+	// TODO: Rollback of non-leaf phases may currently not be implemented
```

Comment:
Nit: "Rollback of non-leaf phases may currently not be implemented" -> Rollback of non-leaf phases is not supported. 

Although I think this get's a bit strange in gravity, because there is a rollback command that can rollback entirely. So may be difficult to convey that this applies specifically here.